docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -809,6 +811,40 @@ SomeOtherBuildItem secondBuildStep() {
 }
 ----
 
+[id='validation-error-build-item']
+===== Validation Error build items
+
+They represent build items with validation errors that make the build fail.
```

Comment:
We should probably add a note that ``ValidationErrorBuildItem``s are actually consumed during initialization of the CDI container so that it's clear that it might cause build cycles.

Reply:
Updated